virtcontainers/interfaces.go

```diff
@@ -20,17 +20,23 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
-// VC is the Virtcontainers interface
-type VC interface {
+// VCManager is the Virtcontainers sandbox management interface
+type VCManager interface {
```

Comment:
Why do we need this new interface ? Is there a benefit ? I am asking cause otherwise I would prefer keep things simple when not needed.

Reply:
It is not strictly needed but it make the interface cleaner and we do want a clean interface IMHO.

Reply:
The separation also makes it clear that the rest of the APIs in `VC` interface will be wrappers of `VCSandbox`.

Reply:
Well I don't understand the clear separation here. Why do we have `StartSandbox()` into the `VCManager` and `StopSandbox()` into the `VC` interface then ?

Reply:
Because we didn't define VCSandbox.Stop() :(

I'll drop this commit to move it out of the API change's way.